public/app/core/components/PageActionBar/PageActionBar.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { PureComponent } from 'react';
-import { FilterInput } from '../FilterInput/FilterInput';
+import { FilterInput } from '@grafana/ui/src/components/FilterInput/FilterInput';
```

Comment:
@mckn Do you think it would make sense to make the FilterInput a public component?

Reply:
I exposed the FilterInput under @grafana/ui and cleaned up the imports added by the VSCode refactor